TESTING.md

```diff
@@ -124,6 +124,10 @@ This test suite is based on [BackstopJS](https://garris.github.io/BackstopJS) pl
 
 Documentation available [here](https://github.com/garris/BackstopJS#backstopjs)
 
+### Requirements
+ * A sample Mosaico custom configured template.
+ * A sample Unscheduled mailing draft with all the three steps forms prefilled.
```

Comment:
also, like above, provide path and any additional info so that the dev is not lost